core/pom.xml

```diff
@@ -73,49 +87,55 @@
-<!--                            <includes>.*</includes>-->
-<!--                            <inputSchema>public</inputSchema>-->
-<!--                        </database>-->
-<!--                        <target>-->
-<!--                            <packageName>eu.fasten.core.data.metadatadb.codegen</packageName>-->
-<!--                            <directory>src/main/java</directory>-->
-<!--                        </target>-->
-<!--                    </generator>-->
-<!--                </configuration>-->
-<!--            </plugin>-->
+            <!--            jOOQ code generation plugin-->
+            <!--            <plugin>-->
+            <!--                <groupId>org.jooq</groupId>-->
+            <!--                <artifactId>jooq-codegen-maven</artifactId>-->
+            <!--                <version>3.12.3</version>-->
+            <!--                <executions>-->
+            <!--                    <execution>-->
+            <!--                        <goals>-->
+            <!--                            <goal>generate</goal>-->
+            <!--                        </goals>-->
+            <!--                    </execution>-->
+            <!--                </executions>-->
+            <!--                <dependencies>-->
+            <!--                    <dependency>-->
+            <!--                        <groupId>org.postgresql</groupId>-->
+            <!--                        <artifactId>postgresql</artifactId>-->
+            <!--                        <version>9.4.1208</version>-->
+            <!--                    </dependency>-->
+            <!--                </dependencies>-->
+            <!--                <configuration>-->
+            <!--                    <jdbc>-->
+            <!--                        <driver>org.postgresql.Driver</driver>-->
+            <!--                        <url>jdbc:postgresql:postgres</url>-->
+            <!--                        <user>postgres</user>-->
+            <!--                        <password>pass123</password>-->
+            <!--                    </jdbc>-->
+            <!--                    <generator>-->
+            <!--                        <database>-->
+            <!--                            <name>org.jooq.meta.postgres.PostgresDatabase</name>-->
+            <!--                            <includes>.*</includes>-->
+            <!--                            <inputSchema>public</inputSchema>-->
+            <!--                        </database>-->
+            <!--                        <target>-->
+            <!--                            <packageName>eu.fasten.core.data.metadatadb.codegen</packageName>-->
+            <!--                            <directory>src/main/java</directory>-->
+            <!--                        </target>-->
+            <!--                    </generator>-->
+            <!--                </configuration>-->
+            <!--            </plugin>-->
```

Comment:
Do we still need this part of the config? Instead of commenting it out, future us could also just look it up in the repository history.

Reply:
We are using it to generate JOOQ classes whenever the DB schema changes. So it better be left commented out for the sake of convenience.

Reply:
Ok, makes sense